From ab0f74f557a596ef8975370ddea087f9a939e518 Mon Sep 17 00:00:00 2001
From: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
Date: Tue, 5 Dec 2023 11:28:48 -0600
Subject: [PATCH 1/1] post_code: Increase the maximum number of post codes to
 8K
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit

Increases the maximum number of post codes to 8192 entries.
Removes the print statement that flooded the console.

Signed-off-by: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
---
 src/post_code.cpp | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/src/post_code.cpp b/src/post_code.cpp
index b96c7c5..483aced 100644
--- a/src/post_code.cpp
+++ b/src/post_code.cpp
@@ -24,8 +24,7 @@
 #define POST_TXT_PATH "/var/lib/phosphor-post-code-manager/host0"
 #define PS_STR_BUFF_SIZE 128
 #define POST_TXT_FILE "-hex.txt"
-#define MAX_POST_CODE_PER_BOOT  (6000)
-#define ERR_LOG_LIMIT (100)
+#define MAX_POST_CODE_PER_BOOT  (8192)

 PostCodeDataHolder* PostCodeDataHolder::instance = 0;

@@ -145,12 +144,9 @@ void PostCode::savePostCodes(postcode_t code)
     // We'll discard newer, repeating POST code entries when it exceeds limit.
     if (postCodes.size() >= MAX_POST_CODE_PER_BOOT)
     {
-        // print once every 100 entries, to avoid spam
-        if ((postCodes.size() % ERR_LOG_LIMIT) == 0)
-        {
-            std::cerr << "Maximum POST entries limit reached."
-                      << "Discarding new entries." << std::endl;
-        }
+        // Do nothing
+        // Discard Post codes
+
     }
     else
     {
--
2.17.1

